### PR TITLE
Add ITS online cluster task

### PR DESCRIPTION
**Pull request by JianLIUhep**

### CODE REVIEW

File: Modules/ITS/include/ITS/ITSClusterTask.h [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+#include <TH1.h>
+#include <TH2.h>
+#include <DataFormatsITSMFT/ROFRecord.h>
+#include <Framework/DataProcessorSpec.h>
+#include <DataFormatsITSMFT/TopologyDictionary.h>
+#include <ITSBase/GeometryTGeo.h>
+#include <DetectorsBase/GeometryManager.h>
+#include <DataFormatsITSMFT/ClusterTopology.h>
+#include <ITSMFTReconstruction/ChipMappingITS.h>
+#include <ITSBase/GeometryTGeo.h>
```
[the remark this answers is not in the document]
Comment (JianLIUhep) [reply Modules/ITS/include/ITS/ITSClusterTask.h]: done.

File: Modules/ITS/include/ITS/ITSClusterTask.h [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+class TH1D;
+class TH2D;
+
+using namespace std;
+using namespace o2::quality_control::core;
+using namespace o2::framework;
+using namespace o2::itsmft;
+using namespace o2::header;
```
[the remark this answers is not in the document]
Comment (JianLIUhep) [reply Modules/ITS/include/ITS/ITSClusterTask.h]: ok, they were removed from the header file

**Comment by JianLIUhep** [comment]
More cleanup for the other ITS tasks will be done in the next PR.

**Review by knopers8 — APPROVED** [review]
Thanks! I am also so picky about these headers, because it has influence on the total build time of QC and I am trying to keep it minimal.

**Pull request merged**